Composer/packages/client/__tests__/utils/dialogUtil.test.js

```diff
@@ -9,9 +9,6 @@ import {
   updateRegExIntent,
```

Comment:
is it possible to convert this to typescript now?

Reply:
yes we can, there is a lib named `ts-jest`

Reply:
we can migrate all test files to *.test.ts in a separate PR